atlassian_jwt_auth/contrib/tests/aiohttp/test_public_key_provider.py

```diff
@@ -21,7 +21,7 @@ def _get_session(self):
 
         resp = session.get.return_value
         resp.headers = CIMultiDict({"content-type": PEM_FILE_TYPE})
-        resp.text.return_value = 'i-am-a-public-key'
+        resp.text = CoroutineMock(return_value='i-am-a-public-key')
```

Comment:
`resp.text` is a coroutine (https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientResponse.text).